golang/testsuite/testsuite_impl/example_testsuite.go

```diff
@@ -36,6 +37,7 @@ func (suite ExampleTestsuite) GetTests() map[string]testsuite.Test {
 			suite.datastoreServiceImage,
 			suite.apiServiceImage,
 		),
+		"waitForEndpointAvailabilityTest": wait_for_endpoint_availability_test.NewWaitForEnpointAvailabilityTest(suite.datastoreServiceImage),
```

Comment:
We'll want to move this down into the set of tests that are only activated upon Kurtosis dev mode. Rationale: users with the free version of Kurtosis only get 3 tests, so if we put this here then all bootstrapped repos will fail saying "You have 4 tests, but only 3 are allowed with the free version"

Reply:
Yes, you are right! even I was commenting on the lines in order to execute it. But users doesn't have to do this.